AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoObserver.swift

```diff
@@ -92,4 +92,7 @@ import Foundation
     ///
     /// - Parameter sources: Array of video sources that are unavailable
     func remoteVideoSourcesDidBecomeUnavailable(sources: [RemoteVideoSource])
+    
+    /// Called on the main thread when video capacity status is updated.
+    func cameraSendAvailabilityDidChange(available: Bool)
```

Comment:
Do we need to add Parameter in comment as well? it will be used to generate docs.

Reply:
Added

Reply:
parameter should be `available`